src/map/battle.cpp

```diff
@@ -8390,8 +8390,11 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 						break;
 					case SU_CN_METEOR:
 					case SU_CN_METEOR2:
-						skillratio += 100 + 100 * skill_lv + sstatus->int_ * 5; // !TODO: Confirm INT bonus
-						RE_LVL_DMOD(100);
+						skillratio += -100 + 200 + 100 * skill_lv;
+						if (status_get_lv(src) > 99) {
+							skillratio += sstatus->int_ * 5;
+							RE_LVL_DMOD(100);
+						}
```

Comment:
```suggestion
						}
						RE_LVL_DMOD(100);
```
I understand the intention behind it, but lets leave it like that.